### PR TITLE
[TW] Translation Finish of Basics Section

### PR DESCRIPTION
**Pull request by lithiumpie**
@doomspork @cizixs 
zh-tw Translation 11.Documentation_12.Testing_13. Comprehensions_14.Strings_15.Custom Mix Tasks_16. IEx Helpers

### CODE REVIEW

**Comment by eksperimental** [comment]
TW is not a valid language code. I highly suggest to use zh-tw as the code, and rename and move to a zh-tw dir

**Comment by eksperimental** [comment]
We may also want to add to instructions that when adding a.new language extract the language code from a link, maybe a wikipedia article.

File: tw/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Elixir School
+redirect_from: /
```
Comment (doomspork) [inline tw/index.md]: We need to remove these `redirect_from:` 👍

File: _data/locales/tw.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+table_of_contents: 目錄
+share: 分享本頁面
+sections:
+  home: Home
+  basics: 基礎
+  advanced: 進階
+  specifics: 特色
+  libraries: 函式庫
+version_messages:
+  outdated: "Some contents of this translation may be outdated."
```
Comment (doomspork) [inline _data/locales/tw.yml]: @lithiumpie did you want to translate these phrases?

Comment (lithiumpie) [reply _data/locales/tw.yml]: Thanks for informing me of it. I will submit the translation later. 👍

File: tw/index.md (hunk 1 above)
Comment (lithiumpie) [reply tw/index.md]: Sure. I already known this issue from #1282 . I will remove all the `redirect_from` from my every translation so far. 👍

**Comment by doomspork** [comment]
@lithiumpie there seems to be a conflict but once that is resolved I'll happily merge 👍

**Comment by cizixs** [comment]
@doomspork Seems to me changes are made according to review comments, can we merge this now?

**Comment by doomspork** [comment]
Thank you thank you thank you @lithiumpie and @cizixs 👍

**Pull request merged**